test/serializer/optimized-functions/optimize_conflict.js

```diff
@@ -0,0 +1,12 @@
+function outer() {
+  function inner() {
+    return 42;
+  }
+  if (global.__optimize) __optimize(inner);
+  return inner;
+}
+if (global.__optimize) __optimize(outer);
+
+global.inspect = function() {
```

Comment:
Why does this test have the word "conflict" in its name?
Is the inner function actually being optimized? I don't see any assertions that optimization actually happens. What is this test testing?